package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Sat Nov 10 21:05:05 UTC 2018 - hellcp@opensuse.org
+
+- Switch installation UI for openSUSE to sidebar (boo#1088785) 
+- 42.3.99.33
```

Comment:
Not implying is wrong or right, but this versioning schema looks odd. Seems to be somehow related to the Leap version... but frozen in the 42.3 times. :thinking:

Reply:
As Tumbleweed actually does not have any version, what about using date in this case? Like `20181113`? `42.3.xx.xx` looks strange to me...

Reply:
BTW the TW `os-release` contains the date version as well so it would really make sense to also use it here. And as `20181113` > `42.3.x` the switch would not make any troubles.